src/style/themes.ts

```diff
@@ -31,28 +31,38 @@ const typography: DefaultTheme["typography"] = {
     lineHeight: 1.5,
     fontWeight: 400,
   },
-  formLabel: {
+  infoHeading: {
     fontSize: "1rem",
     lineHeight: 1.5,
-    fontWeight: 700,
+    fontWeight: 600,
   },
-  formInput: {
+  infoSubHeading: {
     fontSize: "1rem",
     lineHeight: 1.5,
     fontWeight: 400,
   },
+  formLabel: {
+    fontSize: "0.75rem",
+    lineHeight: 1.5,
+    fontWeight: 600,
+  },
+  formInput: {
+    fontSize: "1.5rem",
```

Comment:
Should be `fontSize: "1.125rem",`, no?

![Screenshot 2021-08-07 at 09 41 05](https://user-images.githubusercontent.com/86911296/128592798-6c12c7ab-fd46-4279-bf81-0b44cb5e71e5.png)

Reply:
form input reflects the senderAmount number. senderToken (WETH) as of now is a button that opens the modal; Should we create a Button in the typography section to reflect the 1.125rem font size there? 

Would love to get your opinion on that. @piersss

Reply:
Oh yeah, you're totally right. I was working on the token search input, and confused that input with this one.

Reply:
I do think the button text should probably have a typography component, although not really critical because there's only one place to edit it.